src/store/reducers/block.ts

```diff
@@ -0,0 +1,11 @@
+import { ADD_SINGLE_BLOCK } from '../actions';
+import {BlocksEntity} from "../../helpers/Blocks";
+
+export const block = (state: BlocksEntity[] = [], action) => {
```

Comment:
This is not an array. 
```suggestion
export const block = (state: BlocksEntity = {}, action) => {
```